server/models/CheckPointmodel.js

```diff
@@ -0,0 +1,30 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var CheckPointCollectionModel = new Schema({
+  checkPointPost: {
+    type: String
+  },
+  checkPointAddress: {
+    type: String
+  },
+  checkPointZipcode: {
+      type: Number
+  },
+  checkPointCity: {
+    type: String
+  },
+  checkPointState: {
+    type: String
+  },
+  checkPointDate: {
```

Comment:
You've got two fields for the date, date and time, and neither of them use the native `Date` storage. I'd recommend using just one field and https://docs.mongodb.com/manual/reference/bson-types/#date